src/applications/vaos/new-appointment/redux/actions.js

```diff
@@ -893,13 +890,10 @@ export function checkCommunityCareEligibility() {
     try {
       // Check if user registered systems support community care...
       const siteIds = selectSystemIds(state);
-      const parentFacilities = await getOrganizations({ siteIds });
-      const ccSites = await getSitesSupportingVAR(
-        parentFacilities.map(parent => getSiteIdFromOrganization(parent)),
-      );
-      const ccEnabledSystems = parentFacilities.filter(parent =>
-        ccSites.some(site => site.id === getSiteIdFromOrganization(parent)),
-      );
+      const parentFacilities = await fetchParentLocations({ siteIds });
+      const ccEnabledSystems = await fetchCommunityCareSupportedSites({
```

Comment:
I thought it made sense to move some of this logic into a service, since we're drifting toward having most application/domain logic in those services files, rather than redux.